scripts/run_e2e_tests.py

```diff
@@ -265,21 +311,61 @@ def run_tests(args):
                 'PORTSERVER_ADDRESS': common.PORTSERVER_SOCKET_FILEPATH,
             }))
 
-        stack.enter_context(servers.managed_webdriver_server(
-            chrome_version=args.chrome_driver_version))
-
-        proc = stack.enter_context(servers.managed_protractor_server(
-            suite_name=args.suite,
-            dev_mode=dev_mode,
-            debug_mode=args.debug_mode,
-            sharding_instances=args.sharding_instances,
-            stdout=subprocess.PIPE))
-
-        print(
-            'Servers have come up.\n'
-            'Note: If ADD_SCREENSHOT_REPORTER is set to true in '
-            'core/tests/protractor.conf.js, you can view screenshots of the '
-            'failed tests in ../protractor-screenshots/')
+        if args.suite == 'full':
+            stack.enter_context(servers.managed_webdriver_server(
+                chrome_version=args.chrome_driver_version))
+
+            print('Protractor suites are starting to run')
+            proc = stack.enter_context(servers.managed_protractor_server(
+                suite_name=args.suite,
+                dev_mode=dev_mode,
+                debug_mode=args.debug_mode,
+                sharding_instances=args.sharding_instances,
+                stdout=subprocess.PIPE))
+
+            print('WebdriverIO suites are starting to run')
+            proc = stack.enter_context(servers.managed_webdriverio_server(
+                suite_name=args.suite,
+                dev_mode=dev_mode,
+                debug_mode=args.debug_mode,
+                chrome_version=args.chrome_driver_version,
+                sharding_instances=args.sharding_instances,
+                stdout=subprocess.PIPE))
+
+        elif args.suite in SUITES_MIGRATED_TO_WEBDRIVERIO:
+            proc = stack.enter_context(servers.managed_webdriverio_server(
+                suite_name=args.suite,
+                dev_mode=dev_mode,
+                debug_mode=args.debug_mode,
+                chrome_version=args.chrome_driver_version,
+                stdout=subprocess.PIPE))
+
+            print(
+                'Servers have come up.\n'
+                'Note: You can view screenshots of failed tests '
+                'in ../webdriverio-screenshots/')
+
+        elif args.suite in SUITES_STILL_IN_PROTRACTOR:
+            stack.enter_context(servers.managed_webdriver_server(
+                chrome_version=args.chrome_driver_version))
```

Comment:
Can we move this out of `if-else`?

Reply:
No, this is only needed for protractor tests.